ui/js/controllers/main.js

```diff
@@ -508,10 +508,13 @@ treeherderApp.controller('MainCtrl', [
         };
 
         const getNewReloadTriggerParams = function () {
-            return _.pick(
-                $location.search(),
-                ThResultSetStore.reloadOnChangeParameters,
-            );
+            const locationSearch = $location.search();
+            return ThResultSetStore.reloadOnChangeParameters.reduce((acc, prop) => {
+                                                                            if (locationSearch[prop]) {
```

Comment:
This could be shorter with:

    return ThResultSetStore.reloadOnChangeParameters.reduce((acc, prop) => (
      locationSearch[prop] ? { ...acc, [prop]: locationSearch[prop] } : acc
    ), {});

Would you try this approach on this and the other case and see how that works?